Makefile

```diff
@@ -23,16 +23,16 @@ PROJECT = $(shell git rev-parse --show-toplevel)
 INCLUDE = -I$(PROJECT) -I$(PROJECT)/mbedtls/include
 
 # Set our standard C++ compiler flags
-CXXFLAGS = -g -std=c++20 -fPIC -DSQLITE_ENABLE_NORMALIZE $(BEDROCK_OPTIM_COMPILE_FLAG) -Wall -Werror -Wformat-security  -Wno-error=deprecated-declarations $(INCLUDE)
+CXXFLAGS = -g -std=c++20 -gdwarf-4 -stdlib=libc++ -I/usr/include/c++/v1 -fPIC -DSQLITE_ENABLE_NORMALIZE $(BEDROCK_OPTIM_COMPILE_FLAG) -Wno-vla-cxx-extension -Wno-unqualified-std-cast-call -Werror -Wformat-security  -Wno-error=deprecated-declarations $(INCLUDE) #-Wall -Werror
```

Comment:
`-gdwarf-4` is here because otherwise clang outputs newer dwarf formats (version 5) than gnu ld will understand. We should be able get rid of this if we use clang's `lld` linker. For now, this outputs the older format.